client/src/components/App.js

```diff
@@ -24,16 +26,22 @@ const App = (props) => {
     fetchCurrentUser();
   }, []);
 
+  let greeting = "Hello from react";
+  if (currentUser) {
+    greeting += `, ${currentUser.userName}`;
+  }
```

Comment:
haha something tells me we really dont want this message at all. I would just delete the greeting entirely